openapi/code/service.go

```diff
@@ -304,6 +304,7 @@ func (svc *Service) newMethod(verb, path string, params []openapi.Parameter, op
 		Verb:              strings.ToUpper(verb),
 		Path:              path,
 		Request:           request,
+		ListingRequest:    svc.withPaginationFieldsRemoved(request, op.Pagination),
```

Comment:
I think this is backwards. Listing request should have pagination fields, and Request should not.

Reply:
sure, we can do that, it'll remove a couple of IFs from templates. But then it's not an opt-in feature for the SDK template. if this is not an opt-in feature, we can skip `ListingRequest` altogether.